service/src/main/java/bio/terra/workspace/amalgam/landingzone/azure/LandingZoneApiDispatch.java

```diff
@@ -54,15 +54,11 @@ public class LandingZoneApiDispatch {
   private static final String AZURE_BATCH_ACCOUNT_RESOURCE_TYPE = "Microsoft.Batch/batchAccounts";
 
   private final LandingZoneService landingZoneService;
-  private final WorkspaceService workspaceService;
```

Comment:
`LandingZoneApiDispatch` should not depend on `WorkspaceService`. It appears to have been added as a convenience but causes an unresolved circular dependency in this new code. Removed.